managed-ledger/src/test/java/org/apache/bookkeeper/client/PulsarMockBookKeeper.java

```diff
@@ -249,6 +249,14 @@ void checkProgrammedFail() throws BKException, InterruptedException {
         }
     }
 
+    synchronized boolean checkReturnEmptyLedger() {
+        if (emptyLedgerAfter-- == 0) {
```

Comment:
nit: sometimes I feel putting `--` operation and `=` operation in same line is hard for maintenance. 
```suggestion
        emptyLedgerAfter--;
        if (emptyLedgerAfter == 0) {
```

Reply:
Sure, will change though in this case the decrement has to be done after the comparison